src/nimlsppkg/baseprotocol.nim

```diff
@@ -31,20 +31,14 @@ proc formFrame*(data: JsonNode): string =
 proc sendJson*(s: Stream | AsyncFile, data: JsonNode) {.multisync.} =
   var frame = newStringOfCap(1024)
   toUgly(frame, data)
-  when s is Stream:
-    s.sendFrame(frame)
-  else:
-    await s.sendFrame(frame)
```

Comment:
This breaks `sendJson` as it is able to accept both a `Stream` and an `AsyncFile` but now id doesn't have a code-path for `Stream` any longer..

Reply:
It works fine for me.
`AsyncFile` is passed when it runs async, `Stream` is passed when it doesn't run async.
multisync  strips await when it doesn't run async and `Stream` is passed.

Reply:
Oh sorry, didn't realise this was now in a multisync proc.